scripts/build/build.ps1

```diff
@@ -12,7 +12,7 @@ $ErrorActionPreference="Stop"
 
 . "$PSScriptRoot\..\common\_common.ps1"
 
-_ "$RepoRoot\scripts\build\generate-version.ps1"
+. "$RepoRoot\scripts\build\generate-version.ps1"
```

Comment:
Can you break up this change? There is WAY too much going on here for a single PR!